examples/using-gatsby-script/README.md

```diff
@@ -0,0 +1,23 @@
+# Using gatsby-script
```

Comment:
Not sure wether we should have dedicated `using-gatsby-script` site in our examples (maintainace problem - we have quite a bit of example sites there that are severly outdated, and not sure if adding more like that is good idea).

Reply:
That's true, we could probably get away with example snippets in the documentation pages and skip the example site altogether